pylds/base.py

```diff
@@ -72,28 +73,55 @@ def check_if_points_escape_box(u, box_BOUNDARIES):
     """
     x, y = u.T
     # Escape condition
-    box_x_min, box_x_max = box_BOUNDARIES[0]
-    box_y_min, box_y_max = box_BOUNDARIES[1]
+    box_x_min, box_x_max = box_boundaries[0]
+    box_y_min, box_y_max = box_boundaries[1]
     u_indices = (x >= box_x_min) & (x <= box_x_max) & (y >= box_y_min) & (y <= box_y_max)
     
     return u_indices
 
+def lag_des(u, v, p_value = 0.5):
+    """
+    Vector field equation for Lagrangian descriptor.
+
+    Parameters
+    ----------
+    v : ndarray, shape(n,2)
+        Vector field at given point.
+            
+    p_value : float
+        Exponent in Lagrangian descriptor definition.
+
+    Returns
+    -------
+    LD : ndarray, shape(n,1)
+        Vector field for Lagrangian descriptor dimension.
+    """
+    if p_value == 0:
+        LD = np.abs(u[:,1]*v[:,0])
+    elif p_value>0:
+        LD = np.sum(np.abs(v)**p_value, axis=1)
+    else:
+        LD=np.zeros(len(u[:,0]))
+    return LD
```

Comment:
TWO THINGS:
1. Like for variable names, function names should be explicit, instead of using abbreviations, for consistency in style.
`lag_des` should be written as `lagrangian_descriptor`

2. Although constant, the function is missing the timestep scaling factor, `F = dt**(1 - p_norm)`. See old definition of `accumulate_lagrangian_descriptor`. If you consider this is needless, indicate this in the docstrings of the function.

3. Is the definition of the LD function action-based? Previously in `accumulate_lagrangrian_descriptor`, I had implemented the definition of the M function according to `p_norm`, as indicated in this reference https://doi.org/10.1016/j.cnsns.2015.02.022  Equations 2-4

Reply:
The inverted Duffing is a typo and if you look through the notebook, I cleaned it up quite significantly.

I'll address 1, no problem.

I do not understand 2. Please explain.

3. The M function is still in. The action-based descriptor is new, cf. the pull request and our discussion last week. Please read the pull request and docstrings carefully before resolving to guess work.

Reply:
And now that I look at it, the vector field for the  inverted Duffing ocillator is called HamSN1D_inverted. I'll change that too.

Reply:
> And now that I look at it, the vector field for the inverted Duffing ocillator is called HamSN1D_inverted. I'll change that too.

That's fine. Run the code so that the graphical output is displayed, please.